resources/Explo Development-414318fa79b8.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
A private key such as this should never be committed to a public repo. The key will now need to be retracted.